pkgs/by-name/ba/bash-language-server/package.nix

```diff
@@ -45,19 +45,20 @@ stdenv.mkDerivation (finalAttrs: {
     runHook postBuild
   '';
 
+  # will be fixed in a later commit
+  dontCheckForBrokenSymlinks = true;
+
   installPhase = ''
     runHook preInstall
 
-    pnpm --offline \
-      --frozen-lockfile --ignore-script \
-      --filter=bash-language-server \
-      deploy --prod $out/lib/bash-language-server
+    mkdir -p $out/{bin,lib/bash-language-server}
+    cp -r {node_modules,server} $out/lib/bash-language-server/
```

Comment:
Hmm how many packages are you aware of that have to do this instead of running `pnpm deploy` or `pnpm install`? Maybe it'd be worth documenting this in `doc/languages-frameworks/javascript.section.md`..

Reply:
All pnpm packages that use `pnpm_9` or higher use some of this "ad-hoc" copying instead of `deploy`. Some examples are: etherpad-lite, n8n, tailwindcss-language-server

Yes, documenting it would be amazing, or even putting parts of it in the config hook, but the best would be solving it upstream so we can use `deploy`, I'll hope to find some time for it, feel free to open a tracking issue for now :)